pkg/handler/collector/github/github.go

```diff
@@ -82,7 +88,7 @@ func NewGithubCollector(opts ...Opt) (*githubCollector, error) {
 	if len(g.assetSuffixes) == 0 {
```

Comment:
In `NewGithubCollector` default to `isRelease: true`, so that it will behave as it did before the change if the option is not provided.

Reply:
Done.